crates/mun_hir/src/ty/snapshots/tests__invalid_unary_ops.snap

```diff
@@ -0,0 +1,15 @@
+---
+source: crates/mun_hir/src/ty/tests.rs
+expression: "fn bar(a: int, b: bool) {\n    a = !a; // not is not implemented for int\n    b = -b; // neg is not implemented for bool\n}"
```

Comment:
This does not seem to match with the expected output. There are no errors reported.